src/The_Cartridge_Header.md

```diff
@@ -1,65 +1,75 @@
 # The Cartridge Header
 
-An internal information area is located at $0100-014F in each cartridge.
-It contains the following values:
+Each cartridge contains a header, located at the addresses `$0100`-`$014F`.
+The cartridge header contains the following information:
 
-## 0100-0103 - Entry Point
+## 0100-0103 – Entry Point
 
-After displaying the Nintendo Logo, the built-in boot procedure jumps to
-this address ($0100), which should then jump to the actual main program
-in the cartridge. Usually this 4 byte area contains a `nop` instruction,
-followed by a `jp $0150` instruction. But not always.
+After displaying the Nintendo logo, the built-in boot procedure jumps to
+this address (`$0100`), which should then jump to the actual main program
+in the cartridge. Usually (but not always) this 4-byte area contains a `nop`
+instruction followed by a `jp $0150` instruction.
 
-## 0104-0133 - Nintendo Logo
+## 0104-0133 – Nintendo Logo
+
+These bytes contain a bitmap of the Nintendo logo that is displayed
+when the Game Boy is powered on. The hex dump of this bitmap is:
 
-These bytes define the bitmap of the Nintendo logo that is displayed
-when the Game Boy gets turned on. The hex dump of this bitmap is:
 ```
- CE ED 66 66 CC 0D 00 0B 03 73 00 83 00 0C 00 0D
- 00 08 11 1F 88 89 00 0E DC CC 6E E6 DD DD D9 99
- BB BB 67 63 6E 0E EC CC DD DC 99 9F BB B9 33 3E
+CE ED 66 66 CC 0D 00 0B 03 73 00 83 00 0C 00 0D
+00 08 11 1F 88 89 00 0E DC CC 6E E6 DD DD D9 99
+BB BB 67 63 6E 0E EC CC DD DC 99 9F BB B9 33 3E
 ```
-The Game Boy's boot procedure verifies the content of this bitmap
-(after it has displayed it), and LOCKS ITSELF UP if these bytes are
-incorrect. A CGB verifies only the first half ($18 bytes of) the bitmap, but
-others (for example a Game Boy pocket) verify all $30 bytes, as does the
+
+The Game Boy's boot procedure **first displays** the logo and **then verifies**
```

Comment:
why the bold usage here?

Reply:
I found the fact that the logo is _first_ displayed and only _then_ verified a bit surprising, so I guess I wanted to emphasize that. But I agree with @ISSOtm's comment - there's no point in going into the details of the power-up sequence in this document, since that is outside its scope. @ISSOtm would you mind making the change?